frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java

```diff
@@ -4,14 +4,20 @@
 
 @AutoValue
 public abstract class PlaceVisitModel {
+  public static enum UserMark {
```

Comment:
Sorry I missed this earlier, but it would be preferable to have an "UNKNOWN" (or similar) default state so that we never use null to represent an unset user mark. This actually won't work as it is since UserMark isn't marked as `@Nullable`.

Reply:
I incorporated the UNKNOWN into the other pull request